library/core/src/slice/mod.rs

```diff
@@ -3042,15 +3048,14 @@ impl<T> [T] {
     /// Median of Medians using Tukey's Ninther for pivot selection, which guarantees linear runtime
     /// for all inputs.
     ///
-    /// It is typically faster than stable sorting, except in a few special cases, e.g., when the
-    /// slice is nearly fully sorted, where `slice::sort` may be faster.
-    ///
```

Comment:
Why remove these sections, are they no longer accurate?

Reply:
These sections make sense for `sort_unstable` for which I originally wrote that part. But for `select_nth_unstable` there is currently no stable counterpart, even then that recommendation is based on properties not relevant for partial sorting. In short this is a copy paste mistake 😇